patient-service/src/main/java/com/raf/si/patientservice/model/VaccinationCovid.java

```diff
@@ -0,0 +1,47 @@
+package com.raf.si.patientservice.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Getter
+@Setter
+@Entity
+public class VaccinationCovid {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "vaccine_id", nullable = false)
+    private Vaccine vaccine;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "health_record_id", nullable = false)
+    private HealthRecord healthRecord;
+
+    @Column
+    private Date vaccinationDate = new Date(System.currentTimeMillis());
+
+    @OneToOne(mappedBy = "vaccination", fetch = FetchType.EAGER)
+    private ScheduledVaccinationCovid scheduledVaccinationCovid;
+
+    @Column
+    private String doseReceived = "1";
```

Comment:
Stavi ovo da bude int, proverio sam sa Emom

Reply:
Upravo promenio, tho stavio sam da je long. Nadam se da to nije prob.